src/pug/includes/head.pug

```diff
@@ -10,9 +10,11 @@ meta(name="description" content="swag opportunities for developers")
 link(rel="icon" type="image/png" href="/assets/img/logo.png")
 link(rel="apple-touch-icon" size="128x128" href="/assets/img/logo.png")
 
-link(rel="stylesheet" href=bustedAssets.css)
-link(rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mobius1-selectr@2.4.13/dist/selectr.min.css")
+link(rel="stylesheet" inline=true href=bustedAssets.css)
+link(rel="stylesheet" data-href="https://cdn.jsdelivr.net/npm/mobius1-selectr@2.4.13/dist/selectr.min.css")
 
 noscript
 	style.
 		.buttons {display: none}
+
+	link(rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mobius1-selectr@2.4.13/dist/selectr.min.css")
```

Comment:
In fact, I've just seen that we hide the "buttons", including Selectr on noscript. So this can be dropped from here.